### PR TITLE
Feature/#40 Implement loader on auth page

### PR DESCRIPTION
**Pull request by MangriMen**

### CODE REVIEW

**Comment by MangriMen** [comment]
[Implement loader on auth page](https://trello.com/c/90LlQWYu/40-implement-loader-on-auth-page)

File: src/ducks/auth/selectors.ts [hunk 1]
```diff
@@ -1,5 +1,9 @@
 import { RootState } from 'ducks/store';
 
+export function getIsLoading(state: RootState) {
```
Comment (StepanovRoma) [inline src/ducks/auth/selectors.ts]: Selectors are named like selectSelectedField

**Review by StepanovRoma — CHANGES_REQUESTED** [review]

**Review by StepanovRoma — APPROVED** [review]

**Review by FFreestyler — APPROVED** [review]

File: src/components/pages/AuthPage.tsx [hunk 2]
```diff
@@ -100,11 +112,28 @@ const AuthPage: FC = () => {
...
-            <Register changeComponentType={changeComponentType} />
-          )}
+          <Box
+            flexGrow="1"
+            display="flex"
+            justifyContent="center"
+            alignItems="center"
+            minHeight="21rem"
+          >
+            {isLoading ? (
```
Comment (IHiFiveI) [inline src/components/pages/AuthPage.tsx]: Chaining ternary is usualy a bad practice, so may be would be better use there `&&` and split it into two checks? Like `isLoading && <CircularProgress />`, `!isLoading && ...`

File: src/ducks/user/index.ts [hunk 3]
```diff
@@ -14,4 +14,4 @@ export const userSlice = createSlice({
   },
 });
 
-export { getUser };
+export { selectUser as getUser };
```
Comment (IHiFiveI) [inline src/ducks/user/index.ts]: Why a rename on export?

Comment (MangriMen) [reply src/ducks/user/index.ts]: Oh wait. VSCode auto changed export after rename.

**Pull request merged**